### DISCUSSION TITLE
dev编译后在微信小程序有问题，dev后的真机没有问题，build没有问题

### DISCUSSION THREAD

**Discussion by tangyh001** [NervJS/taro#16229] [Q & A]
按照网上方法关闭了持久化缓存，但是看配置是默认关闭的。

![image](https://github.com/user-attachments/assets/b3f4fcd7-297c-4e1f-bde1-12970adcf54c)

**Comment by tangyh001** [comment]
刚刚从github拉下的内容。

**Reply by honlyHuang** [reply]
从哪里拉的？

**Reply by tangyh001** [reply]
github拉的，选择webpack5，不过现在解决了，编译方式需要改成："dev:weapp": "taro build --ttype ype weapp --watch --env production",
